src/buildercore/core.py

```diff
@@ -75,37 +74,14 @@ class NoMasterException(Exception):
 # sns
 #
 
-def _set_raw_subscription_attribute(sns_connection, subscription_arn):
```

Comment:
the good old days of monkeypatching the SDK :hammer:

Reply:
pfft - I've been wanting to delete that ever since I first saw it. Like an itch you can't scratch.